databay/base_planner.py

```diff
@@ -49,7 +49,9 @@ def add_links(self, links:Union[Link, List[Link]]):
             links = [links]
 
         for link in links:
-            assert isinstance(link, Link)
+            if not isinstance(link, Link):
+                raise TypeError("link is not an instance of Link()")
```

Comment:
I'd suggest we try to be a little more verbose on what is the error. So I'd suggest appending this string as something like:

```python
raise TypeError(f"Provided link is not an instance of Link(), found: {link}")
```

This way we're more specific that we're talking about the link that was provided by the user rather than that it was received in some other way (although this is not the best use case for this improvement), while we also tell user which object exactly caused the issue.

Reply:
Great suggestion, yeah I was thinking I'd like it to be more detailed.